modules/xmpp/strophe.emuc.js

```diff
@@ -224,12 +224,16 @@ export default class MucConnectionPlugin extends ConnectionPluginListenable {
         const visitors = $(iq).find('>visitors[xmlns="jitsi:visitors"]');
         const response = $(iq).find('promotion-response');
 
-        if (visitors.length && response.length
-            && String(response.attr('allow')).toLowerCase() === 'true') {
-            logger.warn('Redirected back to main room.');
-
-            this.xmpp.eventEmitter.emit(
-                CONNECTION_REDIRECTED, undefined, visitors.attr('focusjid'), response.attr('username'));
+        if (visitors.length && response.length) {
+            if (String(response.attr('allow')).toLowerCase() === 'true') {
+                logger.warn('Redirected back to main room.');
```

Comment:
Would it be clearer to log "promotion request accepted" and add a similar message when it's rejected?